### ISSUE TITLE
Selected State with City Screen

### ISSUE THREAD

**Issue by geekanamika** [UdacityAndroidDevScholarship/explore-india#79]
**Summary:**
Create the code/layout for the following mockup provided.

**Mockups**
![image](https://user-images.githubusercontent.com/16922933/39718547-37e2fde8-5254-11e8-9776-40a45f3912d3.png)

**Estimated time**
48 hours

**Comment by arunkbabu90** [comment]
Can I take over this issue?

**Comment by vvkvivekl** [comment]
@arunkbabu90  Go on. reply on it today, or else it will be assigned to someone else tomorrow and try to close it within 2 days.

**Comment by arunkbabu90** [comment]
Yep I'm taking over this issue & I'll fix it as soon as possible @vvkvivekl
